### DISCUSSION TITLE
Setup dev environment for vue-query

### DISCUSSION THREAD

**Discussion by kouts** [DamianOsipiuk/vue-query#164] [Q&A]
Hello @DamianOsipiuk and thanks again for this great project, as a follow up to https://github.com/DamianOsipiuk/vue-query/discussions/104 I would also like to setup a dev environment so that I can familiarize myself with the codebase. 
What would be the recommended setup in order to develop vue-query?

**Comment by DamianOsipiuk** [comment]
1. Clone repository
2. `npm install`
3. `npm link`
4. `npm run watch`
5. Open new terminal
7. `cd examples/basic`
8. `npm install`
9. `npm link vue-query`
10. Uncomment a line in `vite.config`
11. `npm run dev`

Also install official Vue devtools browser addon.

**Comment by kouts** [comment]
I'll try that, thanks @DamianOsipiuk
